pinot-common/src/main/java/org/apache/pinot/common/function/scalar/JsonFunctions.java

```diff
@@ -93,8 +95,7 @@ public static Object jsonPath(Object object, String jsonPath) {
    * Extract object array based on Json path
    */
   @ScalarFunction
-  public static Object[] jsonPathArray(Object object, String jsonPath)
-      throws JsonProcessingException {
+  public static Object[] jsonPathArray(Object object, String jsonPath) {
     if (object instanceof String) {
       return convertObjectToArray(PARSE_CONTEXT.parse((String) object).read(jsonPath, NO_PREDICATES));
```

Comment:
We should check return value before calling `convertObjectToArray()`. Currently it will return `new Object[] {null}` which is incorrect

Reply:
Good catch.

Reply:
I fixed this by handling null in `convertObjectToArray`